pkg/cmd/pulumi/state_move.go

```diff
@@ -146,6 +146,21 @@ func (cmd *stateMoveCmd) Run(
 	}
 	sourceSnapshot.Resources = sourceSnapshot.Resources[:i]
 
+	// Create a root stack if there is none
+	rootStack, err := stack.GetRootStackResource(destSnapshot)
+	if err != nil {
+		return err
+	}
+	if rootStack == nil {
+		projectName, ok := dest.Ref().Project()
+		if !ok {
+			return errors.New("failed to get project name of source stack")
+		}
+		rootStack = stack.CreateRootStackResource(
+			dest.Ref().Name().Q(), tokens.PackageName(projectName))
+		destSnapshot.Resources = append(destSnapshot.Resources, rootStack)
```

Comment:
Shouldn't this always be prepended? Root stack should be the first resource.

Reply:
Yeah, I think prepending makes sense.  Done :heavy_check_mark: